addons/zylann.editor_debugger/dock.gd

```diff
@@ -172,13 +173,6 @@ func _on_Tree_item_selected() -> void:
 	_select_node()
 
 
-func _on_Tree_item_mouse_selected(position: Vector2, mouse_button_index: int) -> void:
```

Comment:
Why remove this method? It handles a signal connected in `dock.tscn`.

Reply:
Somehow I missed that! I will put it back.